packages/grafana-ui/src/themes/mixins.ts

```diff
@@ -0,0 +1,21 @@
+import { GrafanaTheme } from '@grafana/data';
+
+export function cardChrome(theme: GrafanaTheme): string {
```

Comment:
Are we planning on using this styling more places then in the `NewsPanel`? If not I would probably move it to a file located close the the component.

Reply:
yes, this we use card styles in many places (in sass), but this the emotion equivalent to a sass mixing :)

Reply:
Super, got that! But then it feels like it is placed in the correct place since it will be globally used. Looks good! :)

Reply:
@mckn we want to intoduce some reusable pieces of styles as we see more needs for that as there is more emotion code in the codebase